Sources/RemoteMessaging/Matchers/AppAttributeMatcher.swift

```diff
@@ -20,7 +20,59 @@ import Foundation
 import Common
 import BrowserServicesKit
 
-public struct AppAttributeMatcher: AttributeMatching {
+#if os(iOS)
+public typealias AppAttributeMatcher = MobileAppAttributeMatcher
+#elseif os(macOS)
+public typealias AppAttributeMatcher = DesktopAppAttributeMatcher
+#endif
```

Comment:
`installedMacAppStore` is an "app" attribute, so we're adding platform-specific structs for app attribute matching.